x-pack/plugins/fleet/server/services/epm/registry/index.ts

```diff
@@ -272,15 +265,12 @@ async function getPackageInfoFromArchiveOrCache(
   archivePath: string
 ): Promise<ArchivePackage> {
   const cachedInfo = getPackageInfo({ name, version });
-
   if (!cachedInfo) {
     const { packageInfo } = await generatePackageInfoFromArchiveBuffer(
       archiveBuffer,
       ensureContentType(archivePath)
     );
-    // set the download URL as it isn't contained in the manifest
-    // this allows us to re-download the archive during package install
-    setPackageInfo({ packageInfo: { ...packageInfo, download: archivePath }, name, version });
+    setPackageInfo({ packageInfo, name, version });
```

Comment:
caching the `archivePath` is not needed anymore?

Reply:
Yes we don't re-download the archive during install anymore (that was a bug) so this fix isnt needed.